### PR TITLE
Add automcomplete=off to invite/edit user form

### LINKED ISSUE #21023: Add new user: password manager might change new users username
## Context
When creating a new user it will prompt for your password, when you fill this in with the build in macOS password manager it will change the username of the new user to the current username.

I made a screen recording to better illustrate the issue. As you can see underneath the op up the username gets updated
[PIeYkcFHdkw](https://www.youtube.com/watch?v=PIeYkcFHdkw)

## Expected Behavior
When I fill in my password is should keep the new users username

## Possible Solution
It seems like the fields are linked to each other. A password field usually has a username field, so either the fields should be better scooped. An extra username field should be above the password field, disabled and pre-filled in or the password prompt should be on a new page instead of a popup.

## Steps to Reproduce (for Bugs)

1. Create a new user
2. Click "invite user"
3. Have your password be filled in by a password manager. 

## Your Environment
Matomo 4.15.0
PHP 8.2
Safari Version 16.5.1 (18615.2.9.11.7)
macOS 13.4.1 (22F82)

### PR DESCRIPTION
**Pull request by sgiehl**
### Description:

fixes #21023

### Review

* [ ] [Functional review done](https://developer.matomo.org/guides/pull-request-reviews#functional-review-done)
* [ ] [Potential edge cases thought about](https://developer.matomo.org/guides/pull-request-reviews#potential-edge-cases-thought-about) (behavior of the code with strange input, with strange internal state or possible interactions with other Matomo subsystems)
* [ ] [Usability review done](https://developer.matomo.org/guides/pull-request-reviews#usability-review-done) (is anything maybe unclear or think about anything that would cause people to reach out to support)
* [ ] [Security review done](https://developer.matomo.org/guides/security-in-piwik#checklist)
* [ ] [Wording review done](https://developer.matomo.org/guides/pull-request-reviews#translations-wording-review-done)
* [ ] [Code review done](https://developer.matomo.org/guides/pull-request-reviews#code-review-done)
* [ ] [Tests were added if useful/possible](https://developer.matomo.org/guides/pull-request-reviews#tests-were-added-if-usefulpossible)
* [ ] [Reviewed for breaking changes](https://developer.matomo.org/guides/pull-request-reviews#reviewed-for-breaking-changes)
* [ ] [Developer changelog updated if needed](https://developer.matomo.org/guides/pull-request-reviews#developer-changelog-updated-if-needed)
* [ ] [Documentation added if needed](https://developer.matomo.org/guides/pull-request-reviews#documentation-added-if-needed)
* [ ] Existing documentation updated if needed

### CODE REVIEW

**Review by bx80 — APPROVED** [review]
Looks good to me, the autocomplete=off tag is added to the username input element and at least on Firefox the username is not updated when typing the admin password. I can't vouch for the MacOS password manager though. Perhaps @michalkleiner can test that?

File: plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue [hunk 1]
```diff
@@ -111,6 +112,7 @@
               @update:model-value="theUser.password = $event; isPasswordModified = true"
               uicontrol="password"
               name="user_password"
+              autocomplete="off"
```
Comment (michalkleiner) [inline plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue]:
```suggestion
              autocomplete="new-password"
```

Comment (michalkleiner) [reply plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue]: Would this be a better option for the password?

Comment (sgiehl) [reply plugins/UsersManager/vue/src/UserEditForm/UserEditForm.vue]: makes sense.

**Pull request merged**